XCUITests/ToolbarTest.swift

```diff
@@ -96,6 +96,28 @@ class ToolbarTests: BaseTestCase {
         XCTAssertEqual(value as? String, "", "The url has not been removed correctly")
     }
 
+    //Check that that after scrolling on a page, the URL bar is hidden. Tapping one on the status bar will reveal the URL bar, tapping again on the status will scroll to the top
+    func testRevealToolbarWhenTappingOnStatusbar(){
+        //Workaround when testing on iPhone. If the orientation is in landscape on iPhone the tests will fail.
+        if !iPad() {
+            XCUIDevice.shared.orientation = UIDeviceOrientation.portrait
+        }
+        waitforExistence(app/*@START_MENU_TOKEN@*/.buttons["HomePanels.TopSites"]/*[[".otherElements[\"Panel Chooser\"]",".buttons[\"Top sites\"]",".buttons[\"HomePanels.TopSites\"]"],[[[-1,2],[-1,1],[-1,0,1]],[[-1,2],[-1,1]]],[0]]@END_MENU_TOKEN@*/)
+        navigator.openURL(website1["url"]!, waitForLoading: true)
+        waitforExistence(app/*@START_MENU_TOKEN@*/.buttons["TabLocationView.pageOptionsButton"]/*[[".buttons[\"Page Options Menu\"]",".buttons[\"TabLocationView.pageOptionsButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/)
+        let statusbarElement = app.statusBars.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
+        XCTAssertTrue(statusbarElement.isHittable)
+        app.swipeUp()
+        let hiddenStatusbarElement = app.statusBars.children(matching: .other).element.children(matching: .other).element(boundBy: 0)
+        let pageActionMenuButton = app/*@START_MENU_TOKEN@*/.buttons["TabLocationView.pageOptionsButton"]/*[[".buttons[\"Page Options Menu\"]",".buttons[\"TabLocationView.pageOptionsButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/
```

Comment:
This `pageActionMenuButton` could be defined after line 106 and so used in that waitfor in L107 as a variable...